content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,154 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+### Attaching Files
+
+Attaching files to input elements or dropping them over the page is available
+in Cypress 10.0. Read the [`.attachFile()` API docs](/api/commands/attachfile)
+for more information on how this works and how to use it.
+
+The
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin has been deprecated in favor of `.attachFile()` built into Cypress.
+There's guidance below on how to migrate from the
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin to Cypress's built-in attachFile command.
+
+#### Quick guide
+
+The argument signature is different for `.attachFile()` than the `cypress-file-upload` plugin. You can follow the steps below for each argument in order to migrate:
+
+In the first argument:
+- `filePath`: Prefix value with `fixtures/'. Rename to `contents`.
```

Comment:
Was formatting awkwardly. I believe the second line needed to be removed, right? In favor of the next line? 

```suggestion
- `filePath`: Prefix value with `fixtures/`.
```

Reply:
I added the missing tick.

In cypress-flie-upload, you could only use one at a time - providing both together was an error. So there are really two cases: Rename filePath to contents, *or* rename fileContent to contents. I'm open to better ways to explain that in the migration guide, but I think it would be clear enough when a user sees their code

```
{
  filePath: 'foo.json',
  encoding: 'utf-8',
}
```

that it turns into `{contents: 'fixture/foo.json'}` and that when they have

```
{
  fileContent: Blob.atob([91, 10]),
  mimeType: 'text/plain',
}
```

turns into `{content: Buffer.from([91, 10])}`.

I can't provide specific guidance on the Blob -> Buffer change because it's very application specific.

Reply:
Got it. thx!